controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -2231,7 +2237,7 @@ private HttpResponse cancelDeploy(String tenantName, String applicationName, Str
                 return;
             }
 
-            ChangesToCancel cancel = ChangesToCancel.valueOf(choice.toUpperCase());
+            ChangesToCancel cancel = ChangesToCancel.valueOf(choice.replaceAll("-", "_").toUpperCase());
```

Comment:
:raised_eyebrow:

Reply:
We use hyphenated rest api paths, afaik, and that can't be an enum name.